algorithm_catalog/sentinel1_stats.json

```diff
@@ -0,0 +1,106 @@
+            "sentinel-1",
+            "statistics"
+        ],
+        "language": {
+            "code": "en-US",
+            "name": "English (United States)"
+        },
+        "languages": [
+            {
+                "code": "en-US",
+                "name": "English (United States)"
+            }
+        ],
+        "contacts": [
+            {
+                "name": "Pratichhya Sharma",
+                "position": "Researcher",
+                "organization": "VITO",
+                "links": [
+                    {
+                        "href": "https://www.vito.be/",
+                        "rel": "about",
+                        "type": "text/html"
+                    },
+                    {
+                        "href": "https://github.com/pratichhya",
+                        "rel": "about",
+                        "type": "text/html"
+                    }
+                ],
+                "contactInstructions": "Contact via VITO",
+                "roles": [
+                    "principal investigator"
+                ]
+            },
+            {
+                "name": "VITO",
+                "links": [
+                    {
+                        "href": "https://www.vito.be/",
+                        "rel": "about",
+                        "type": "text/html"
+                    }
+                ],
+                "contactInstructions": "SEE WEBSITE",
+                "roles": [
+                    "processor"
+                ]
+            }
+        ],
+        "themes": [
+            {
+                "concepts": [
+                    {
+                        "id": "Active Remote Sensing"
+                    },
+                    {
+                        "id": "Sentinel-1 GRD"
+                    }
+                ],
+                "scheme": "https://gcmd.earthdata.nasa.gov/kms/concept/c8fe757b-b530-4d67-a553-e4903f4430a5"
+            }
+        ],
+        "formats": [
+            "GeoTiff",
+            "netCDF"
+        ],
+        "license": "CC-BY-4.0"
+    },
+    "linkTemplates": [],
+    "links": [
+        {
+            "rel": "openeo-process",
+            "type": "application/json",
+            "title": "openEO Process Definition",
+            "href": "https://raw.githubusercontent.com/ESA-APEx/apex_algorithms/openeo_udp/sentinel1_stats/sentinel1_stats.json"
+        },
+        {
+            "rel": "service",
+            "type": "application/json",
+            "title": "CDSE openEO federation",
+            "href": "https://openeofed.dataspace.copernicus.eu"
+        },
+        {
+            "rel": "example",
+            "type": "application/json",
+            "title": "Example output",
+            "href": "https://openeo.dataspace.copernicus.eu/openeo/1.1/jobs/j-2410035111a348ef96f6778e4be39101/results/assets/M2UyNGUyNTEtMmU5YS00MzhmLTkwYTktZDQ1MDBlNTc2NTc0/195d6d8857a37478fb4b2036c9063340/openEO.nc?expires=1728548058"
```

Comment:
_Documentation:  The most common way to do this is to store
it in a public git repository. Tagging a release is a good way to ensure that the link remains stable._

@jdries Is there already a dedicated repo or storage we should use? I want to avoid making use of VITO specific storage, as it will confuse external users